README.md

```diff
@@ -417,3 +421,17 @@ function promiseUnresolvable(arg, callback) {
 
 }
 ```
+
+## Module Long Term Support Policy
+
+This module adopts the [
+Module Long Term Support (LTS)](http://github.com/CloudNativeJS/ModuleLTS) policy,
+ with the following End Of Life (EOL) dates:
+
+| Version | Status          | Published | EOL      |
+| ------- | --------------- | --------- | -------- |
+| 4.x     | Active LTS      | Aug 2018  | Dec 2020 |
+| 3.x     | End-of-Life     | May 2018  | Dec 2019 |
+| 2.x     | End-of-Life     | May 2018  | Dec 2019 |
```

Comment:
Actually, we say the status is "end-of-life", but EOL date is Dec 2019 (in the future). I am proposing to change this table as follows:

```markdown
| Version | Status          | Published | EOL      |
| ------- | --------------- | --------- | -------- |
| 4.x     | Active LTS      | Aug 2018  | Dec 2020 |
| 3.x     | End-of-Life     | May 2018  | Aug 2019 |
| 2.x     | End-of-Life     | May 2018  | Aug 2019 |
```

I picked March 2019 because it's 12 months after Aug 2018 when 4.x was published. IIRC our LTS policy, we promise 6 months in Active LTS + 6 months in Maintenance LTS, that's 12 months.